Lib/test/test_textwrap.py

```diff
@@ -1076,5 +1077,24 @@ def test_first_word_too_long_but_placeholder_fits(self):
         self.check_shorten("Helloo", 5, "[...]")
 
 
+class WideCharacterTestCase(BaseTestCase):
+    def setUp(self):
+        def text_len(text):
+            n = 0
+            for c in text:
+                if unicodedata.east_asian_width(c) in ['F', 'W']:
```

Comment:
Very minor: the list here could be a set

Reply:
fixed